index.html

```diff
@@ -174,6 +174,18 @@ <h2>Skills</h2>
         </div>
       </div>
       <button type="button" class="button-get-started">Start collaboration</button>
+
+      <footer class="footer">
```

Comment:
- [ ] Please about myself section title should be wrapped in hx tag since semantically `hx` is used for titles, also note that you should use only the h1 element in the project.